modules/pool/invest/components/PoolInvestProportional.tsx

```diff
@@ -131,12 +131,15 @@ export function PoolInvestProportional({ onShowPreview }: Props) {
                         {investOptions.map((option, index) => {
                             const tokenOption = selectedInvestTokens[index];
                             const amount = inputAmounts[tokenOption.address];
+                            const tokenOptions = option.tokenOptions.filter(
```

Comment:
Wouldn't do it like this, rather still show both options but disable ones with 0 balance, or just still show both

Reply:
Commit reverted.